src/current/v23.1/advanced-changefeed-configuration.md

```diff
@@ -70,30 +70,51 @@ Use the following workflow to enable `MuxRangefeed`:
+A changefeed aggregates checkpoints across all ranges, and once the timestamp on all the ranges advances, the changefeed can then [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). In the context of changefeeds, `kv.closed_timestamp.target_duration` affects how old the checkpoints will be, which will determine the latency before changefeeds can consider the history of an event complete.
+
+As a result, adjusting `kv.closed_timestamp.target_duration` can affect changefeeds encountering latency **and** changefeeds causing foreground SQL latency. In clusters running large-scale workloads, it may be helpful to:
+
+- **Decrease** the value for a lower changefeed emission latency — that is, the time from committing a write until the changefeed emits that event.
+- **Increase** the value to reduce the potential impact of changefeeds on SQL latency. This will lower the resource cost of changefeeds, which can be especially important for workloads with tables in the TB range of data.
+
+It is important to note that a changefeed at default configuration does not checkpoint more often than once every 30 seconds. When you create a changefeed with [`CREATE CHANGEFEED`]({% link {{ page.version.version }}/create-changefeed.md %}), you can adjust this with the [`min_checkpoint_frequency`]({% link {{ page.version.version }}/create-changefeed.md %}#min-checkpoint-frequency) option.
+
+#### `kv.rangefeed.closed_timestamp_refresh_interval`
+
+**Default:** `3s`
+
+This setting controls the interval at which [closed timestamp]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#closed-timestamps) updates are delivered to [rangefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) and in turn emitted as a [changefeed checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}).
+
+Increasing the interval value will lengthen the delay between each checkpoint, which will increase the latency of changefeed checkpoints, but reduce the impact on SQL latency due to [overload]({% link {{ page.version.version }}/admission-control.md %}#use-cases-for-admission-control) on the cluster. This happens because every range with a rangefeed has to emit a checkpoint event with this `3s` interval. As an example, across 1 million ranges that would result in 330,000 events per second, which would use more CPU resources.
+
+If you are running changefeeds at a large scale and notice foreground SQL latency, we recommend increasing this setting.
+
+{{site.data.alerts.callout_info}}
+The `kv.closed_timestamp.side_transport_interval` cluster setting controls how often the closed timestamp is updated (with a default of `200ms`). Although the closed timestamp is updated every `200ms`, CockroachDB will only emit an event across the rangefeed containing the closed timestamp value every `3s` as per the `kv.rangefeed.closed_timestamp_refresh_interval` value.
```

Comment:
Right now, I do not have a separate section for `side_transport`. Perhaps it needs one? That is, is it a setting that changefeed users are often tuning to improve latency (either form)?

Reply:
I see that it isn't mentioned under our [closed timestamp docs](https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer#closed-timestamps) either. Might be worth adding a brief section on it separately, and link it from here.

Closed timestamps are normally propagated via Raft commands. But an idle range sees no Raft commands, so it would stall. The side transport is an efficient mechanism to broadcast closed timestamp updates for all idle ranges between nodes.

It affects both follower reads and changefeeds/rangefeeds. Users have historically used the side transport settings to tune the checkpoint interval, instead of `kv.rangefeed.closed_timestamp_refresh_interval`, but the latter is better since it won't affect follower reads.